shipyard-controller/docs/swagger.yaml

```diff
@@ -154,12 +158,24 @@ definitions:
       creationDate:
         description: Creation date of the project
         type: string
+      gitProxyScheme:
+        description: git proxy scheme
+        type: string
+      gitProxyUrl:
+        description: git proxy URL
+        type: string
+      gitProxyUser:
+        description: git proxy user
+        type: string
       gitRemoteURI:
         description: Git remote URI
         type: string
       gitUser:
         description: Git User
         type: string
+      insecureSkipTLS:
```

Comment:
Why do we need to repeat it here? It is also at line 73

Reply:
it was auto-generated by swagger, but we can remove it

Reply:
it's part of different models